test.py

```diff
@@ -83,6 +90,27 @@ def test_path(self):
         # All done, remove graph.
         redis_graph.delete()
 
+    def test_edge(self):
+        edge1 = Edge('john', 'visited', 'japan', properties={'purpose': 'pleasure'})
```

Comment:
An `Edge` object assumes input arguments at positions 0 `src` and 2 `dest` to be of type `Node`